silkworm/rpc/ws/connection.hpp

```diff
@@ -57,6 +57,7 @@ class Connection : public StreamWriter {
     // Methods of StreamWriter interface
     Task<void> open_stream() override { co_return; }
     Task<void> close_stream() override { co_return; }
+    size_t get_capacity() override { return 5 * 1024 * 1024; }
```

Comment:
Better use a constant instead of an hard-coded magic number

Reply:
ok